src/js/facility-locator/containers/VAMap.jsx

```diff
@@ -102,31 +104,45 @@ class VAMap extends Component {
 
     this.updateUrlParams({
       address: currentQuery.searchString,
-      location: `${currentQuery.position.latitude},${currentQuery.position.longitude}`,
     });
     this.props.search(currentQuery);
   }
 
+  renderFacilityMarkers() {
+    const facilities = this.props.facilities;
+
+    return facilities.map(f => {
+      return (
+        <NumberedIcon key={f.id} position={[f.lat, f.long]} number={f.id}>
+          <span>{`Facility ${f.id}: ${f.attributes.name}`}</span>
+        </NumberedIcon>
+      );
+    });
+  }
+
   render() {
   // defaults to White House coordinates initially
     const coords = this.props.currentQuery.position;
     const position = [coords.latitude, coords.longitude];
+    // used for setting a proper map height
+    const mapHeight = window.innerHeight - document.getElementsByClassName('header')[0].clientHeight - 175;
 
     return (
       <div>
-        <div className="columns medium-3">
-          <ResultsPane onSearch={this.handleSearch}/>
+        <div className="columns medium-4">
+          <ResultsPane onSearch={this.handleSearch} maxHeight={mapHeight}/>
```

Comment:
I'll remove this—I set it so that the results are scrollable with the overflowY. Something to think about for both mobile and desktop: have we talked about results pagination?